content/blog/2023/2023-03-08-code-format-with-ktlint.md

```diff
@@ -0,0 +1,134 @@
+## Introduction
+In this tutorial, we are going to learn about a linting tool used by developers while writing code using Kotlin language called Ktlint. Essentially, what Ktlint does is check the styling we have used to write our code if it's standard and also helps us in formatting where we went against the guidelines. This ensures that our code is kept clean, easier to read, understand and maintain.
+
+
+## Introduction
+In this tutorial, we are going to learn about a linting tool used by developers while writing code using Kotlin language called Ktlint. Essentially, what Ktlint does is check the styling we have used to write our code if it's standard and also helps us in formatting where we went against the guidelines. This ensures that our code is kept clean, easier to read, understand and maintain.
+
+{{% github "https://github.com/thombergs/code-examples/tree/master/kotlin" %}}
+
+## What Is Linting
+Linting simply refers to the process of analyzing our code for any potential errors, typos or defects in formatting. This entire process is achieved by using a lint tool/ static code analyzer.
```

Comment:
Maybe refer to the source of the name ... linting originally means "removing dust".